File: FIRE DRILL/PYTHON/task_nine.py
sum_sum = 0
total = 0
number = 4
total_sum = 0
square = 0

while(number <= 2000):
    total += number
    number*=4        

numberTwo = 8
while(numberTwo <= 35000):
    sum_sum += numberTwo
    numberTwo*=8
          
        
total_sum = total + sum_sum
square = total_sum * total_sum

print(square)

File: FIRE DRILL/JAVA/TaskNine.java
public class TaskNine{
    public static void main(String[]args){

        int sum = 0;
        int total = 0;
        int square = 0;
        int totalSum = 0;

        for(int number = 4; number <= 2000; number*=4){
           total += number;
           
         }           

        for(int numberTwo = 8; numberTwo <= 35000; numberTwo*=8){
           sum += numberTwo;
          
        }
        totalSum = total+sum;
        square = totalSum*totalSum; 
        

        System.out.println(square);


    }
}
